tests/e2e-ibc/go.mod

```diff
@@ -0,0 +1,263 @@
+module github.com/kava-labs/kava/tests/interchain
+
+go 1.21.9
+
+replace (
+	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
+	github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
+	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+	github.com/vedhavyas/go-subkey => github.com/strangelove-ventures/go-subkey v1.0.7
+)
+
+require (
+	cosmossdk.io/math v1.3.0
+	github.com/cosmos/ibc-go/v8 v8.1.0
+	github.com/strangelove-ventures/interchaintest/v8 v8.2.0
```

Comment:
Nit: Should these be matched with kava v0.26.x versions?

Reply:
as you mentioned, i think we'll want them to match for conformance testing. i'll note it in a ticket to be updated in a future pr 👍